gbfs.md

```diff
@@ -709,12 +709,12 @@ Field Name | REQUIRED | Type | Defines
-&emsp;&emsp;\-&nbsp;`vehicle_type_id`| Yes | ID | REQUIRED if `vehicle_type_area_capacity` is defined. A `vehicle_type_id`, as defined in `vehicle_types.json`, that may park at the virtual station.
-&emsp;&emsp;\-&nbsp;`count`| Yes | Non-negative integer | REQUIRED if `vehicle_type_area_capacity` is defined. A number representing the total number of vehicles of the corresponding `vehicle_type_id` type that can park within the virtual station.<br /><br />If the virtual station is defined by `station_area`, this is the number that can park within the station area. If `lat`/`lon` is defined, this is the number that can park at those coordinates.
-\-&nbsp;`vehicle_type_dock_capacity` <br/>*(as of v3.0)* | OPTIONAL | Array | This field's value is an array of objects containing the keys `vehicle_type_id` and `count` defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle defined in `vehicle_types.json`.
-&emsp;&emsp;\-&nbsp;`vehicle_type_id`| Yes | ID | REQUIRED if `vehicle_type_dock_capacity` is defined. A `vehicle_type_id`, as defined in `vehicle_types.json`, that may dock at the station.
-&emsp;&emsp;\-&nbsp;`count`| Yes | Non-negative integer | REQUIRED if `vehicle_type_dock_capacity` is defined. The total number of docks at the station, both available and unavailable, that may accept the corresponding vehicle type as defined by its `vehicle_type_id`.
+\-&nbsp;`vehicle_type_area_capacity` <br/>*(as of v3.0)* | OPTIONAL | Array | This field's value is an array of objects containing the keys `vehicle_type_ids` and `count` defined below.  These objects are used to model the parking capacity of virtual stations (defined using the `is_virtual_station` field) for each vehicle type that can be returned to this station.
+&emsp;&emsp;\-&nbsp;`vehicle_type_ids`| Yes | Array | REQUIRED if `vehicle_type_area_capacity` is defined. The `vehicle_type_ids`, as defined in `vehicle_types.json`, that may park at the virtual station.
+&emsp;&emsp;\-&nbsp;`count`| Yes | Non-negative integer | REQUIRED if `vehicle_type_area_capacity` is defined. A number representing the total number of vehicles of the specified `vehicle_type_ids` that can park within the virtual station.<br /><br />If the virtual station is defined by `station_area`, this is the number that can park within the station area. If `lat`/`lon` is defined, this is the number that can park at those coordinates.
+\-&nbsp;`vehicle_docks_capacity` <br/>*(as of v3.0)* | OPTIONAL | Array | This field's value is an array of objects containing the keys `vehicle_type_ids` and `count` defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle that may dock at this station. The total of these values should equal the total `capacity` value.
```

Comment:
Is this "should" a `SHOULD` or just a regular `should`?

Reply:
To avoid any ambiguity, this `should` MUST be replaced by either `SHOULD` or `MUST`.

I think `SHOULD` is the best solution, as "that there may exist valid reasons in particular circumstances to ignore" it (maybe if the capacity is not strictly correlated to the number of docks or if some type of docks overlap)

`MUST`/`SHOULD`: https://datatracker.ietf.org/doc/html/rfc2119

Reply:
Thanks for catching this! I ought to have put "MUST", but am open to revisiting if there are other cases that I haven't considered.

In the case of overlapping dock/space types: this would be represented by including each union of supported vehicle types in a separate array item.

For example (In a system with scooters, bikes, and ebikes):
A station has a total capacity of 10, half of which support all vehicles, and half support only pedal bikes.
There would be two `vehicle_docks_capacity` array items:
- one with _all_ `vehicle_type_ids` and a `count` of 5
- one with just the pedal bike id, and another `count` of 5

This is a break from the previous format, where you would have one array item for each vehicle type:
`vehicle_type_id: pedal_bike, count: 10`, scooter: 5, ebike: 5 etc.,  which would often lead to a different `total_capacity` from the sum of each item in the array, when there are overlapping dock types.

I think capacity should (MUST) be correlated with the total number of spaces for each item in the array, but maybe I'm misunderstanding the situation you're describing, or there's some other case I haven't considered.

Reply:
@tdelmas @testower or others, can you think of a use case where the total number of docks from each type of dock would **not** add up to match the station `capacity`?

Reply:
I dont have any concrete examples, but overlapping dock types seems to be a fairly obvious theoretical case

Reply:
At the risk of repeating @cmonagle's explanation: overlapping dock types _can_ be represented even with the clause total number of docks `MUST` equal the `capacity`:

Indeed, in @cmonagle's example, the total number of docks from each objects _does_ adds up to the`capacity`, even with overlapping dock types (ie: the 5 docks that can accept pedal bikes and ebikes):

station_information.json:
```
"capacity": 10,
"vehicle_docks_capacity": [
  {
    "vehicle_type_ids": ["pedal_bike", "ebike"],
    "count": 5
  },
  {
    "vehicle_type_ids": ["pedal_bike"],
    "count": 5
  }
]
```

The question is: are there concrete or theoretical use cases where a station would have a `capacity` that is different from the number of docks?

Reply:
Thanks 😅 no I can’t think of any examples

Reply:
Thank you all for your participation.

Feel free to open a new issue to continue the discussion if needed. Thanks!